src/sentry/static/sentry/app/views/settings/project/projectServiceHookDetails.jsx

```diff
@@ -138,9 +138,12 @@ export default class ProjectServiceHookDetails extends AsyncView {
           <PanelHeader>{t('Event Validation')}</PanelHeader>
           <PanelBody>
             <PanelAlert type="info" icon="icon-circle-exclamation">
-              Sentry will send the <code>X-ServiceHook-Signature</code> header built using{' '}
-              <code>HMAC(SHA256, [secret], [payload])</code>. You should always verify
-              this signature before trusting the information provided in the webhook.
+              <div>
```

Comment:
How likely do you think it is that we'll have missed adding a div to an existing alert?

Reply:
In order to fix another problem, I've add:

```
display: flex;
flex: 1;
```

and consequently I broke this part of the code..so adding a div fixed the problem...but it seems that `width: 100%` also fixes my problem and we don't need to add extra `divs`..I've updated the code

Reply:
Should this div remain?